terraform-modules/mongodb/config.tf

```diff
@@ -5,8 +5,7 @@ resource "google_storage_bucket_object" "docker-compose" {
 version: '2'
 services:
   mongodb:
-    user: root
-    image: bitnami/mongodb:3.6.13
+    image: bitnami/mongodb:${var.mongodb_image_tag}
```

Comment:
This is interesting...like fundamentally I want to keep pushing this out--why not allow parameterizing the whole image? why not also parameterize  all the env vars? And then you get into a position where you've kind of reimplemented k8s on gcp instances. Not asking for any changes--I think I get why it makes sense this way here--just observing.